oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/OAuth2TokenAttributes.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2002-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.oauth2.core;
+
+import org.springframework.util.Assert;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *
+ * @author Clement Ng
+ *
+ */
+public class OAuth2TokenAttributes {
```

Comment:
It'd be great to have a test class for `OAuth2TokenAttributes`.

Reply:
I was considering it, but wasn't sure what would be tested w/i the test class that wasn't already covered in `OAuth2IntrospectionAuthenticationTokenTests`. Should I pull some attribute-specific tests from `OAuth2IntrospectionAuthenticationTokenTests`?

Reply:
Fair point, we can add tests for this class if and when it becomes more sophisticated.